javascript/chronometer.js

```diff
@@ -1,26 +1,43 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime= 0;
+    this.intervalId= 0;
+
   }
   startClick(callback) {
-    // ... your code goes here
-  }
-  getMinutes() {
-    // ... your code goes here
-  }
-  getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
-  }
+
+this.intervalId = setInterval(() => this.currentTime++;
+callback()
+}, 1000)
+
+}
+
+  getMinutes(currentTime) {
```

Comment:
These functions don't need a parameter currentTime, you are accesing the currentTime information via the this keyword (this.currentTime)